SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -387,17 +254,11 @@ void InterestFactory::AddSystemQuery(Interest& OutInterest, const AActor* InActo
 {
```

Comment:
Can you rename AddSystemQuery ?

Reply:
errrrrrr

Reply:
created a new variable, `FullFrequencyOptional`, used throughout whenever necessary

Reply:
renamed